1907-11-07.xml

```diff
@@ -1704,233 +1724,240 @@
+                    <p>Mohr and Fenderl, 93 " "</p>
+                    <p>G. Riecken, 30 " "</p>
+                    <p>F. Andres, 15 " "</p>
+                    <p>J. Planta and Co., 220 " "</p>
                     <p>3,259 bales cotton</p>
                     <p>Behrend and Co, 537 tons cotton seed</p>
                 </div>
-                <div type="item">
+                <div type="item" status="verified">
                     <p>Telegramme Havas </p>
-                    <p>BUURSR du 6 novembre 1907 COURS DES VALEURS TERME OLOTURE PARIS Route
-                        Francaise 3 of ... </p>
-                    <p>94 92 Dette Egyptienne Unifibe . </p>
-                    <p>101 50 x Extérieur Kepegaal.... . </p>
-                    <p>91 40 Russe consolidé... </p>
-                    <p>75 70 Aotions de Sue... ... ... </p>
-                    <p>4620 - Oredlo Fonctor Egyptlen </p>
-                    <p>676 Onedit Lyonnais ... </p>
-                    <p>1165 Comptoi National d'Brompt Banque Ottomane. Land Bank of Egypt </p>
-                    <p>180 Banque d'Athènes. Banque d'Abyssinie.. Ordis Franco Raption... Ohange sur
-                        Londru </p>
-                    <p>LONDRES Consolides anglais </p>
-                    <p>- - 8 32 % Roomptes-Paris 31%, Londres 6 olo, Berlin 6 o'o </p>
-                    <p>LIITIFIT </p>
-                    <p>104 </p>
+                    <p>BOURSE du 6 novembre 1907 </p>
+                    <p>COURS DES VALEURS TERME, OLOTURE </p>
+                    <p>PARIS </p>
+                    <p>Route Francaise 3 o/o ...  ...  ...  ...  Fr. 94 92</p>
+                    <p>Dette Egyptienne Unifiee ...  ...  ... " 101 50 ex</p>
+                    <p>Extérieur Kepegnal ...  ...  ...  ...  ... " 91 40</p>
+                    <p>Russe consolidé...  ...  ...  ...  ...  ...  " 75 70</p>
+                    <p>Aotions de Sue... ... ...  ...  ... " 4620 – </p>
+                    <p>Credit Fonctor Egyptien ... ... ... " 676 – </p>
+                    <p>Credit Lyonnais ... ... ... ... ... " 1145 – </p>
+                    <p>Comptoi National d'Brompt <!-- unclear --> ...  ... "678 – </p>
+                    <p>Banque Ottomane  ...  ...  ...  ...  ... " 688 .– </p>
+                    <p>Land Bank of Egypt  ...  ...  ...  ... " 180 – </p>
+                    <p>Banque d'Athènes.  ...  ...  ...  ...  ... " 104 – </p>
+                    <p>Banque d'Abyssinie..  ...  ...  ...  ... " – – </p>
+                    <p>Ordis Franco-Egyptian... ...  ...  ... " 105 – </p>
+                    <p>Ohange sur Londres ...  ...  ...  ... " 25 19</p>
+                    <p>LONDRES </p>
+                    <p>Consolides anglais  ...  ...  ...  ...  ... (euro) 8 32 1/3</p>
+                    <p>Roomptes-Paris 31 1/2%, Londres 6 %, Berlin 6 ½  ... </p>
                 </div>
-                <div type="item">
+                <div type="item" status="verified">
```

Comment:
Use the template: https://dig-eg-gaz.github.io/issues/templates/#primes-des-contrats